documentation/book/con-custom-resources-example.adoc

```diff
@@ -50,13 +52,15 @@ spec: <2>
 <2> The specification for this CRD, including the group (domain) name, the plural name and the supported schema version, which are used in the URL to access the API of the topic. The other names are used to identify instance resources in the CLI. For example, `kubectl get kafkatopic my-topic` or `kubectl get kafkatopics`.
 <3> The shortname can be used in CLI commands. For example, `kubectl get kt` can be used as an abbreviation instead of `kubectl get kafkatopic`.
 <4> The information presented when using a `get` command on the custom resource.
-<5> openAPIV3Schema validation provides validation for the creation of topic custom resources. For example, a topic requires at least one partition and one replica.
+<5> The current status of the CRD.
+<6> openAPIV3Schema validation provides validation for the creation of topic custom resources. For example, a topic requires at least one partition and one replica.
 
 NOTE: You can identify the CRD YAML files supplied with the {ProductName} installation files, because the file names contain an index number followed by ‘Crd’.
 
 Here is a corresponding example of a `KafkaTopic` custom resource.
 
 .Kafka topic custom resource
+//SHOULD WE ADD THE STATUS PROPERTY TO THIS EXAMPLE?
```

Comment:
@scholzj - Please let me know what you think here.

Reply:
Yes, I think we should add a status example. Do you have one from Tom? Or should I get one for you?

Reply:
Please can you provide one? Thanks.

Reply:
Ok, this seems to be the status I get in KafkaTopic:

```yaml
status:
  conditions:
  - lastTransitionTime: "2019-08-20T11:37:00.706Z"
    status: "True"
    type: Ready
  observedGeneration: 1
```

Reply:
Nice! Thank you.

Reply:
Does it return a `reason` for the last transition?

Reply:
I think `reason` will be there only in case of some error.